apps/designer/app/designer/features/style-panel/sections/layout/shared/flex-grid.tsx

```diff
@@ -44,18 +44,25 @@ export const FlexGrid = ({
 
   return (
     <Grid
+      tabIndex={0}
       css={{
         width: "100%",
         aspectRatio: "1 / 1",
         padding: theme.spacing[4],
         borderRadius: "4px",
         background: theme.colors.loContrast,
-        border: "2px solid currentColor",
+        outlineStyle: "solid",
+        outlineWidth: styleSource === "default" ? 1 : 2,
+        outlineColor: color,
         alignItems: "center",
         gap: theme.spacing[1],
         gridTemplateColumns: "repeat(3, 1fr)",
         gridTemplateRows: "repeat(3, 1fr)",
         color,
+        "&:focus, &:focus-within": {
+          outlineWidth: 2,
+          outlineColor: theme.colors.borderSetFlexUi,
```

Comment:
focus and local colors are almost same. I didn't see the difference at first.

Reply:
Yeah, will comment in figma, its a token

Reply:
Actually it is all clear in figma, we just needed to upgrade colors to tokens